lib/compat/wordpress-5.9/class-wp-theme-json-gutenberg.php

```diff
@@ -1017,9 +1018,16 @@ private static function scope_selector( $scope, $selector ) {
 	private static function get_settings_values_by_slug( $settings, $preset_metadata, $origins ) {
 		$preset_per_origin = _wp_array_get( $settings, $preset_metadata['path'], array() );
 
+		$skip_default_presets =
+			array( 'color', 'duotone' ) === $preset_metadata['path'] &&
+			! _wp_array_get( $settings, array( 'color', 'defaultDuotone' ), true );
```

Comment:
@oandregal Should setting `defaultGradients` or `defaultPalette` to `false` also prevent those classes/variables from generating?

Reply:
I decided that it should be and changed it in 98a67533c9b45620b6742652793f60158ab38e8c